importer_client/python/tools/timesketch_importer.py

```diff
@@ -462,6 +482,34 @@ def main(args=None):
         help=("Path to the file that is to be imported."),
     )
 
+    config_group.add_argument(
+        "--analyzer_names",
+        "--analyzer-names",
+        type=comma_separated_list,
+        action="store",
+        dest="analyzer_names",
+        default=[],
+        help=(
+            "Set of analyzers that we will automatically run right after the "
+            "timelines are uploaded. The input needs to be the analyzers names."
+            "Provided as a comma-separated string."
+        ),
+    )
+
+    config_group.add_argument(
+        "--timeline_ids",
+        "--timeline-ids",
+        type=comma_separated_int_list,
+        action="store",
+        dest="timeline_ids",
+        default=[],
+        help=(
+            "Set of timelines that we will automatically analyze right after the "
+            "timelines are uploaded. The input needs to be the analyzers IDs."
+            "Provided as a comma-separated integer list."
+        ),
+    )
```

Comment:
Do we need the user to provide this information? I don't think the user knows the timeline ID prior to uploading in a typical workflow. 
The upload function returns a timeline object that contains the ID of the just uploaded timeline, so this could be used instead?
Let me know if I'm missing the use-case here.